lib/api/assertions/_enabled.js

```diff
@@ -42,7 +42,7 @@ exports.assertion = function(definition, msg) {
   };
```

Comment:
you also need to rename the filename back.

Reply:
Done ✅